src/api/Shrooms.Premium/Presentation/WebViewModels/ValidationAttributes/DateTimeLessThanPresentDateAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Shrooms.Premium.Presentation.WebViewModels.ValidationAttributes
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    public class DateTimeLessThanPresentDateAttribute : ValidationAttribute
+    {
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        {
+            if (value == null || ((DateTime)value) <= DateTime.UtcNow)
```

Comment:
Does UI send dates in UTC? and maybe better to validate if value is of type DateTime first, as this one can throw exception if attribute was set on some other property

Reply:
It does send UTC. At least everything related with events and profile editing. Also, is that check necessary when the attribute name clearly states that it has to be used with DateTime?